tests/e2e/driver/ebs_csi_driver.go

```diff
@@ -109,6 +109,12 @@ func GetParameters(volumeType string, fsType string, encrypted bool) map[string]
 	if iops := IOPSPerGBForVolumeType(volumeType); iops != "" {
 		parameters["iopsPerGB"] = iops
 	}
+	if iopsTotal := IOPSTotalForVolumeType(volumeType); iopsTotal != "" {
+		parameters["iopsTotal"] = iopsTotal
```

Comment:
I see other fields are not using them, but can you use the constants you defined above? Ie `IopsTotalKey` and `ThroughputKey `